src/inspect.rs

```diff
@@ -78,12 +78,12 @@ impl MetadataPrettyPrinter {
         }
 
         let pretty_annotations = vec![
-            "io.kubewarden.policy.title",
-            "io.kubewarden.policy.description",
-            "io.kubewarden.policy.author",
-            "io.kubewarden.policy.url",
-            "io.kubewarden.policy.source",
-            "io.kubewarden.policy.license",
+            crate::constants::ANNOTATION_POLICY_TITLE,
+            crate::constants::ANNOTATION_POLICY_DESCRIPTION,
+            crate::constants::ANNOTATION_POLICY_AUTHOR,
+            crate::constants::ANNOTATION_POLICY_URL,
+            crate::constants::ANNOTATION_POLICY_SOURCE,
+            crate::constants::ANNOTATION_POLICY_LICENSE,
```

Comment:
By `use crate::constants` we could remove some of the duplication here and use `constants::THE_CONST`

Reply:
fixed